parachain/runtime/src/lib.rs

```diff
@@ -367,6 +367,42 @@ impl pallet_treasury::Config for Runtime {
     type MaxApprovals = MaxApprovals;
 }
 
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+    items as Balance * KINT.one() + (bytes as Balance) * KINT.one()
```

Comment:
I don't know the `pallet_elections_phragmen` pallet, but this looks weird. VotingBondBase is apparently 65 kint, while VotingBondFactor is 32 kint? Some additional comment on what this is exactly would be very welcome

Reply:
Copied the exact setup from Kusama's runtime (since we use the same units) and left additional comments.

Reply:
Ah your new version pf the deposit function looks at least plausible now🙂 guess we'll trust kusama for now

Reply:
Ah your new version pf the deposit function looks at least plausible now🙂